datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -306,3 +310,17 @@ def has_e2e(check):
                     if 'pytest.mark.e2e' in test_file.read():
                         return True
     return False
+
+
+def has_legacy_signature(check):
+    """
+    Validate that the given check does not use the legacy agent signature (contains agentConfig)
+    """
+    for path, _, files in os.walk(get_check_directory(check)):
+        for fn in files:
```

Comment:
`fn`? usually mean `function`

Maybe `file` is more appropriate?

Reply:
I was using `fn` as an abbreviation for "filename", but file makes more sense. Since `file` is a keyword, I changed it to `f`.